src/Go.Windows/Go.Windows.csproj

```diff
@@ -0,0 +1,105 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
nit: can you please revert this before check in? CPS projects are more maintainable.

Reply:
Can we revert later? Can't find any docs about CPS. This should be a trivial difference.